src/frontend/src/components/public/PIAFormTabs/review/ProgamArea/displayProgramArea.tsx

```diff
@@ -37,8 +37,11 @@ const DisplayProgramArea = (props: IDisplayProgramAreaProps) => {
   const canEditProgramAreaReview =
     reviewPagePrivilegeParams?.editProgramAreaReview ?? false;
 
-  const allowUserReviewProgramArea = () => {
-    if (!canEditProgramAreaReview) return false;
+  // Logic Explanation:
+  //   - When this function returns true, it means the readonly.
+  //   - When this function returns false, it means allow editing the review.
```

Comment:
i may add to the comments if you are drafter, etc you will always be in view mode
if you are a reviewer, you can only edit your review etc. 

The processing of what make it true or false can be further elaborated.

Reply:
If you are a drafter you can add a review/edit your own review

Reply:
> i may add to the comments if you are drafter, etc you will always be in view mode if you are a reviewer, you can only edit your review etc.
> 
> The processing of what make it true or false can be further elaborated.

yes right, I will add more elaborated comments in terms of the drafter, reviewer etc.